inc/3rd-party/plugins/cookie-notice.php

```diff
@@ -20,7 +20,7 @@
  * @param $cookies array List of dynamic cookies
  * @return array List of dynamic cookies with the Cookie Notice cookie appended
  */
-function rocket_get_cookie_notice_cookie($cookies) {
+function rocket_get_cookie_notice_cookie( $cookies ) {
```

Comment:
@Tabrisrp I believe you were talking about this and I hope you do not mind the first empty line after opening `{`. I think the empty line improves readability but if that is something that needs changing, please let me know and I will do it that way from here on.

Reply:
👍